java/integTest/src/test/java/glide/cluster/CommandTests.java

```diff
@@ -728,6 +730,10 @@ public static Stream<Arguments> callCrossSlotCommandsWhichShouldFail() {
                 Arguments.of("sinter", null, clusterClient.sinter(new String[] {"abc", "zxy", "lkn"})),
                 Arguments.of(
                         "sunionstore", null, clusterClient.sunionstore("abc", new String[] {"zxy", "lkn"})),
+                Arguments.of(
+                        "sunionstore",
```

Comment:
This string is used for logging only
```suggestion
                        "sunionstore binary",
```
Please update below too